src/index.js

```diff
@@ -20,9 +20,13 @@ module.exports = {
     })
   },
   getGlobalYarnParser: function(yarnPath, cb) {
+    console.log('yarnPath', yarnPath);
     // <yarn>/bin/yarn ==> (src) <yarn>/src/lockfile
     //                     (compiled) <yar>lib/node_modules/yarn/lib/lockfile/parse.js
-    var parser = require(path.resolve(yarnPath, '../../lib/lockfile/parse.js'))
+    var parserPath = path.resolve(yarnPath, '../../lib/lockfile/parse.js')
+    console.log('parserPath', parserPath)
+    var parser = require(parserPath)
+    console.log('parser', parser);
     return cb(null, parser.default)
```

Comment:
Still debugging, I'll get there.  I'm not at my windows machine right now so this was perhaps a little foolish to start now.

You can see the errors here: https://ci.appveyor.com/project/clarkbw/devtools-core-yjdu8/build/1.0.59

I suspect that because we're using a `./packages/module/node_modules/` directory format the relative path selection isn't working as expected.  However I haven't shown that as of yet.